src/main/java/ai/deepcode/javaclient/core/DeepCodeParamsBase.java

```diff
@@ -16,6 +18,7 @@
   // Inner params
   private String loginUrl;
   private String ideProductName;
+  private Supplier<Long> getTimeoutForGettingAnalysesMs;
```

Comment:
@ArtsiomCh, does it make sense to set default here?

Reply:
Don't think so. We use Supplier to let user of our library to choose how that timeout will be dynamically (or not) constructed

Reply:
I see now... have sense, but it's java (not kotlin), here making default param will cause to create second constructor which seems to be a bit overhead...